ci-operator/step-registry/hypershift/mce/dump/hypershift-mce-dump-commands.sh

```diff
@@ -30,5 +30,6 @@ fi
 CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
 /tmp/hs-cli/hypershift dump cluster --artifact-dir=$ARTIFACT_DIR \
 --namespace local-cluster \
+--agent-namespace "local-cluster-${CLUSTER_NAME}" \
```

Comment:
for `--agent-namespace`, we need to verify whether the hosted cluster platform is `agent`
```
oc get hostedclusters -n local-cluster ${CLUSTER_NAME} -ojsonpath="{.spec.platform.type}"
```